pkg/client/create_builder.go

```diff
@@ -361,3 +402,39 @@ func uriFromLifecycleVersion(version semver.Version, os string, architecture str
 
 	return fmt.Sprintf("https://github.com/buildpacks/lifecycle/releases/download/v%s/lifecycle-v%s+linux.%s.tgz", version.String(), version.String(), arch)
 }
+
+func stripTopDirAndWrite(layerReader io.ReadCloser, outputPath string) (*OS.File, error) {
+	file, err := OS.Create(outputPath)
+	if err != nil {
+		return nil, err
+	}
+
+	tarWriter := tar.NewWriter(file)
+	tarReader := tar.NewReader(layerReader)
+	tarReader.Next()
```

Comment:
Why do you need this?

Reply:
Its to make the tarReader skip the first entry, so that the top level directory can be discarded

Reply:
Good, maybe you can just add a comment, you know, for your future self 😄